reference_designs/ipu-xrt/matrix_multiplication/test.cpp

```diff
@@ -188,10 +189,21 @@ int main(int argc, const char *argv[]) {
     memcpy(CVec.data(), bufOut, (CVec.size() * sizeof(C_DATATYPE)));
     if (do_verify) {
       if (verbosity >= 1) {
-        std::cout << "Verifying against reference matmul ..." << std::endl;
+        if (do_verify_stochastic) {
+          std::cout << "Verifying " << verify_stochastic_n_samples
+                    << " random samples against reference matmul ..."
+                    << std::endl;
+        } else {
+          std::cout << "Verifying against reference matmul ..." << std::endl;
+        }
       }
       auto vstart = std::chrono::system_clock::now();
-      errors = matmul_common::verify(M, N, K, AVec, BVec, CVec);
+      if (do_verify_stochastic) {
```

Comment:
Is this reversed?

Reply:
Good catch. As it stands, the meaning of this variable is `do_NOT_verify_stochastic`, and I'm printing the wrong log string (says stochastic when it's doing full verification and vice versa). I confused myself. Will flip the condition. Actual verification is correct though